publication-commons/src/main/resources/features/publish_publication.feature

```diff
@@ -0,0 +1,30 @@
+Feature: Publish a Publication
+
+  Scenario: The User publishes a Publication
+    Given the User wants to publish a Publication
+    When they set the Accept header to "application/json"
+    And they set the Authentication header to a Bearer token with their <credentials>
+    And they request POST /publication/{identifier}/publish
+    Then they receive a response with status code 202
+    And they see that the response Content-type is "application/json"
```

Comment:
Perhaps a location header that corresponds to the location of the published version of the document?

Reply:
For example?

Reply:
I think in this case, just returning the URI for the publication is enough — it is there that the resource will show changes in status